database/migrations/2019_06_21_074115_create_choices_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateChoicesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('choices', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('word_id');
+            $table->text('text');
+            $table->boolean('is_correct')->default(0);
+            $table->timestamps();
+        });
+
+        Schema::table('choices', function(Blueprint $table) {
+            $table->foreign('word_id')->references('id')->on('words');
```

Comment:
@christanplaza 
You can put this together in create function since this is a new migration file. I can just re run `php artisan migrate:fresh`

Reply:
Noted.

Reply:
This is resolved in ec0abbd